server/core/region.go

```diff
@@ -41,6 +41,7 @@ type RegionInfo struct {
 	readKeys        uint64
 	approximateSize int64
 	approximateKeys int64
+	reportInterval  uint64
```

Comment:
Can we use  `pdpb.TimeInterval` directly?

Reply:
OK